custodia.spec

```diff
@@ -191,6 +192,24 @@ cp %{buildroot}/%{_bindir}/custodia-cli %{buildroot}/%{_bindir}/custodia-cli-3
 %endif
 
 
+%pre
+getent group custodia >/dev/null || groupadd -r custodia
+if ! getent passwd custodia >/dev/null; then
+     useradd -r -g custodia -d / -s /sbin/nologin -c "User for custodia" custodia
+     if [ -d %{_sysconfdir}/custodia ]; then
+         chown -R custodia:custodia %{_sysconfdir}/custodia
+     fi
```

Comment:
Why do we need this ?
Shouldn't we just state what the permission should be in install lines ?

If this is for upgrades it should be handled in upgrades only.